cedar-policy-generators/src/abac.rs

```diff
@@ -120,11 +120,38 @@ pub struct ConstantPool {
     string_constants: Vec<SmolStr>,
 }
 
+#[derive(Debug, Clone, Copy)]
+struct BiasedI64(i64);
+
+impl<'a> Arbitrary<'a> for BiasedI64 {
+    fn arbitrary(u: &mut Unstructured<'a>) -> arbitrary::Result<Self> {
+        Ok(gen!(u,
+            1 => std::i64::MAX,
+            1 => std::i64::MIN,
+            8 => <i64 as Arbitrary>::arbitrary(u)?
```

Comment:
more "special" values we might want to bias for include 0, 1, and -1

Reply:
I ended up adding `-1` and `0` because 1 should be easily obtained by mutating 0.